packages/blocks/src/__internal__/content-parser/index.ts

```diff
@@ -4,6 +4,7 @@ import { Slot } from '@blocksuite/store';
 import { marked } from 'marked';
 
 import type { PageBlockModel } from '../../models.js';
+import { getFileFromClipboard } from '../clipboard/index.js';
```

Comment:
This breaks Node compatibility. Any idea to workaround?

Reply:
It may be caused by side effects importing in `packages/blocks/src/__internal__/clipboard/utils.ts`.

I'm trying to extract a new util file that keeps no dependencies on others. Make it can be relied on by other file such as `packages/blocks/src/__internal__/content-parser/index.ts`